### PR TITLE
Ajax switch reqs to post

### PR DESCRIPTION
**Pull request by DanHoganRed**
Resolve issue #27

### CODE REVIEW

**Comment by DanHoganRed** [comment]
I resolved the conflict. I was try to add a border to the table and forgot to remove it.

File: src/main/resources/static/MainPage.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
             sub_perk = $( "#sub_perk" ),
             desc_perk = $( "#desc_perk" )
 
+        if(location.hostname === "localhost" || location.hostname === "127.0.0.1")
+        {
+            cntxPath = "http://localhost:8081";
+        }else
+        {
+            cntxPath = "https://perkmanager.herokuapp.com";
+        }
```
Comment (raymondwu1) [inline src/main/resources/static/MainPage.js:19]: Is there any alternative to this or do we have to do it this way?

**Review by raymondwu1 — APPROVED** [review]
Works on heroku good job

File: src/main/resources/static/MainPage.js (hunk 1 above)
Comment (DanHoganRed) [reply src/main/resources/static/MainPage.js:19]: I added an issue, #28 for me to take a look at doing that.

**Pull request merged**